x-pack/plugins/enterprise_search/public/applications/app_search/components/result_settings/result_settings.test.tsx

```diff
@@ -32,8 +37,10 @@ describe('RelevanceTuning', () => {
     jest.clearAllMocks();
   });
 
+  const subject = () => shallow(<ResultSettings engineBreadcrumb={['test']} />);
```

Comment:
I like this! I think I might start stealing this more. But I also like rerender depending on the test/component 🤔 Haha, I'm probably overthinking it

Reply:
Ah, OK. I hadn't thought of using rerender for this. This is primarily just to DRY up the code a little bit. The `subject` pattern is something we use in Rails tests in ent-search.